packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -287,13 +287,8 @@ contract L2Genesis is Deployer {
     function setProxyAdmin() public {
         // Note the ProxyAdmin implementation itself is behind a proxy that owns itself.
         address impl = _setImplementationCode(Predeploys.PROXY_ADMIN);
-
-        bytes32 _ownerSlot = bytes32(0);
-
-        // there is no initialize() function, so we just set the storage manually.
-        vm.store(Predeploys.PROXY_ADMIN, _ownerSlot, bytes32(uint256(uint160(cfg.proxyAdminOwner()))));
         // update the proxy to not be uninitialized (although not standard initialize pattern)
-        vm.store(impl, _ownerSlot, bytes32(uint256(uint160(cfg.proxyAdminOwner()))));
+        vm.store(impl, bytes32(0), bytes32(uint256(0xdead)));
```

Comment:
todo: double check if this is necessary, this adds complexity to the genesis definition

Reply:
I can't think of why it would be necessary... is there something in storage at that slot on OP Mainnet?